src/dotnet/project.json

```diff
@@ -54,7 +54,8 @@
     "Microsoft.Win32.Registry": {
       "version": "4.0.0-rc3-24201-00",
       "exclude": "compile"
-    }
+    },
+	"System.Net.NetworkInformation": "4.1.0-rc2-*"
```

Comment:
We do not want * versions in our project.json files.  Especial "rc2" versions.  Please use the current CoreFX version.